src/Storages/MergeTree/MergeTreeSettings.h

```diff
@@ -64,6 +64,7 @@ struct Settings;
     M(UInt64, merge_tree_clear_old_broken_detached_parts_ttl_timeout_seconds, 1ULL * 3600 * 24 * 30, "Remove old broken detached parts in the background if they remained intouched for a specified by this setting period of time.", 0) \
     M(UInt64, merge_tree_enable_clear_old_broken_detached, false, "Enable clearing old broken detached parts operation in background.", 0) \
     M(Bool, remove_rolled_back_parts_immediately, 1, "Setting for an incomplete experimental feature.", 0) \
+    M(String, clean_deleted_rows, "never", "Is the Replicated Merge cleanup has to be done automatically at each merge or manually (possible values are \"always\"/\"never\"(default))", 0) \
```

Comment:
I think enum-based setting would be better here, see https://github.com/ClickHouse/ClickHouse/blob/7c94f98f6ab0c163ca3a7bfc79837efd621f0a24/src/Core/SettingsEnums.h